package.json

```diff
@@ -91,7 +91,7 @@
     "@typescript-eslint/eslint-plugin": "^4.33.0",
     "@typescript-eslint/parser": "^4.33.0",
     "audit-ci": "6.6.1",
-    "ava": "^3.12.1",
+    "ava": "^5.3.1",
```

Comment:
There are more recent releases but this is as far as we can take it given our current Node version. Can revisit once we upgrade.